tools/development/k8s/memory/patch_cgroup_driver/apply.sh

```diff
@@ -0,0 +1,11 @@
+#/usr/bin/env sh
+
+set -e
+
+read -p "This script will stop and reconfigure kubelet/docker services, then reboot all k8s worker nodes. Are you sure (y/N)? " CHOICE
+case "$CHOICE" in
+  y|yes|Y|Yes) ;;
+  *) exit ;;
+esac
+
+exec ansible-playbook -vv main.yml
```

Comment:
```suggestion
ansible-playbook -vv main.yml
```
I think it's better to see in process tree what called `ansible-playbook`